kdocker: add page (#6044)
tldr-pages/tldr: pages/linux/kdocker.md

# kdocker

> > Easily dock applications to system tray.

Reviewer: ```suggestion
> Easily dock applications to the system tray.
```

Author: @navarroaxel your a Linux user, what do you think about this

Reviewer 2: I think the suggestion from @bl-ue is correct, the sentence needs the `the`.

> More information: <https://github.com/user-none/KDocker>.

> - Click on a window and send it to system tray:

Reviewer 2: This description is strange, maybe you can use something similar to https://github.com/tldr-pages/tldr/blob/main/pages/common/xkill.md

Author: Would you like to suggest?
If not I'll happily do it :)

Reviewer 2: ```suggestion
- Display a cursor to send a window to system tray when pressing the left mouse button (press any other mouse button to cancel):
```

> - Display a cursor to send a window to system tray when pressing the left mouse button (press any other mouse button to cancel):

Reviewer: ```suggestion
- Display a cursor to send a window to the system tray when pressing the left mouse button (press any other mouse button to cancel):
```

`kdocker`

- Open an application and send it to system tray:

`kdocker {{application}}`

> - Send focused windows to system tray:

Reviewer 2: How do you focus more than 1 window at the same time?
```suggestion
- Send focused window to system tray:
```
Note: This command only works with terminals because you are in the terminal when you press enter :upside_down_face:

Author: Seems to work with my VScode

> - Open an application and send it to system tray:

Reviewer: ```suggestion
- Open an application and send it to the system tray:
```

> - Send focused window to system tray:

Reviewer: ```suggestion
- Send focused window to the system tray:
```

`kdocker -f`

> - Click on a window and send it to system tray with custom icon:

Reviewer 2: ```suggestion
- Display a cursor to send a window to system tray with a custom icon when pressing the left mouse button
```

Reviewer 3: Missing colon in the suggestion there?

Reviewer 2: Yep!

Author: Fixed

Author: The fixer has become the wrong-doer

> - Display a cursor to send a window to system tray with a custom icon when pressing the left mouse button

Author: ```suggestion
- Display a cursor to send a window to system tray with a custom icon when pressing the left mouse button:
```

Reviewer 2: You have a button to edit the file directly:
![Screenshot_20210524_183935](https://user-images.githubusercontent.com/1536740/119410562-6db0a280-bcbf-11eb-86bc-e1f82856bcaa.png)

Reviewer: That's cool; I've never actually used that before!

Author: Fully aware, just quicker for to do it there, since I was already there :laughing:

> - Display a cursor to send a window to system tray with a custom icon when pressing the left mouse button:

Reviewer: ```suggestion
- Display a cursor to send a window to the system tray with a custom icon when pressing the left mouse button:
```

`kdocker -i {{/path/to/icon}}`

> - Open an application and send it to system tray and if focus is lost, minimize it:

Reviewer: ```suggestion
- Open an application and send it to the system tray and if focus is lost, minimize it:
```

> - Open an application and send it to the system tray and if focus is lost, minimize it:

Reviewer 3: Perhaps this will read better?
```suggestion
- Open an application, send it to the system tray and if focus is lost, minimize it:
```

`kdocker -l {{application}}`

> - Print kdocker version:

Reviewer 2: I think we don't need to repeat the command name here
```suggestion
- Print version:
```
